04_setup_ironic.sh

```diff
@@ -12,6 +12,9 @@ source validation.sh
 
 early_deploy_validation
 
+# Account for differences in 1.* and 2.* version reporting.
+PODMAN_VERSION=$(sudo podman version -f json | jq -r '.Version,.Client.Version|strings')
```

Comment:
Shorter version could be ```sudo podman version --format "{{.Version}}"```

Reply:
The format argument to podman doesn't work on some versions.

Reply:
And it's .Client.Version on newer podmans..

Reply:
There's also a .Server.Version but it seems both are supported